core/services/ardupilot_manager/main.py

```diff
@@ -205,10 +205,10 @@ def available_boards() -> Any:
     config = Config(app=app, loop=loop, host="0.0.0.0", port=8000, log_config=None)
     server = Server(config)
 
+    if args.sitl:
```

Comment:
From the commit scope and message, this change is not connected with the scope of the commit, and it's fixing something else.

Reply:
My bad. It's part of fcd4a4cb.